cypress/Dockerfile

```diff
@@ -1,17 +1,20 @@
 ARG NODE_VERSION='20.9.0'
-ARG CYPRESS_VERSION='13.3.1'
+ARG CYPRESS_VERSION='13.5.0'
 
-FROM cypress/base:${NODE_VERSION}
-
-# Update npm to latest version
-RUN npm install -g npm@latest
-
-# Install necessary packages
-RUN apt-get update && apt-get install -y curl unzip && apt-get clean && rm -rf /var/lib/apt/lists/*
+# Using specific versioned tag that includes Node, Cypress and Edge versions
+FROM cypress/browsers:node-20.9.0-chrome-118.0.5993.88-1-ff-118.0.2-edge-118.0.2088.46-1
 
 # Set working directory
 WORKDIR /hmda-frontend
 
+USER root
+RUN apt-get update && apt-get install -y curl nodejs npm microsoft-edge-stable
```

Comment:
Is `nodejs` needed here if Node is already included in the `cypress/browsers` image?